### ISSUE TITLE
Other possible researches using H as an index 

### ISSUE THREAD

**Issue by ellenhsieh** [uchicago-computation-workshop/marc_berman#18]
Since the article mentions that GConn's correlations with fractal scaling may indicate that H is a marker of highly functionally connected brain regions. Also, it says that H is broadly uni-directional, which is more reliable indicator of task-related effort, unlike activation analysis. Therefore, I am wondering if there is any other psychological researches which also use H to index certain status or phenomena.

**Comment by bermanm** [comment]
Yes.  This seems to be a growing area of research and other researchers are using these measures
